.gitignore

```diff
@@ -41,6 +41,10 @@
 !docs/pt-br/index.md
 !docs/pt-br/**/
 !docs/pt-br/**/*.md
+!docs/fa/
```

Comment:
Nicely spotted that we have an allow-list rather than a deny-list, this is to help keep secrets and temp files oiut of the repo 👍🏾